wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,372 @@
+If you import any modules into your project, such as
+[`@metamask/detect-provider`](https://github.com/MetaMask/detect-provider), use a bundler such as
+[Webpack](https://github.com/webpack/webpack) to compile the modules and create an output script
+`dist/main.js`.
+See [Webpack's Getting Started guide](https://webpack.js.org/guides/getting-started/) for more information.
+:::
+
+In the `simple-dapp` directory, create `src` for the source files and `dist` for the distribution files that output from your Webpack builds. 
+Create a JavaScript file in `src`, then create an HTML file in `dist` from the `simple-dapp` root.
+Include the following HTML code example to establish a link to the JavaScript file generated by Webpack:
+
+```jsx title="index.html"
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Simple dapp</title>
+</head>
+<body>
+    <script src="../dist/index.js"></script>  <!-- Links to the JavaScript bundle created by Webpack -->
+</body>
+</html>
+```
+
+#### Install and configure Webpack
+
+Navigate to the root of your `simple-dapp` directory and run the following commands to initialize npm.
+
+```bash 
+npm init -y && npm install --save-dev webpack webpack-cli
+```
+
+Create a `webpack.config.js` in the root of your project with the following configuration:
+
+```jsx title="index.html"
+const path = require('path');
+
+module.exports = {
+    entry: './src/index.js',
+    output: {
+        path: path.resolve(__dirname, 'dist'),
+        filename: 'index.js'
+    },
+    mode: 'development'
+};
+```
+
+Then, edit `package.json` to include a build script:
+
+```json
+"scripts": {
+    "build": "webpack"
+}
+```
+
+When you run `npm run build` from your terminal, Webpack bundles the content in `src/index.js` into `dist/main.js`, which is linked in `index.html`.
+
+### 2. Detect MetaMask
+
+You can use the `@metamask/detect-provider` module to detect the MetaMask Ethereum provider.
+
+#### Install `@metamask/detect-provider`
+
+Use [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) to install
+`@metamask/detect-provider` in your project directory:
+
+```bash
+npm i @metamask/detect-provider
+```
+
+#### Detect the MetaMask provider
+
+The following code example demonstrates how to detect the MetaMask provider in your dapp using `@metamask/detect-provider`:
+
+```jsx title="src/index.js"
+import detectEthereumProvider from "@metamask/detect-provider";
+
+async function setup() {
+    const provider = await detectEthereumProvider();
+
+    if (provider && provider === window.ethereum) {
+        console.log('MetaMask is available!');
+        startApp(provider); // Initialize your dapp with MetaMask.
+    } else {
+        console.log("Please install MetaMask!");
+    }
+}
+
+function startApp(provider) {
+    if (provider !== window.ethereum) {
+        console.error("Do you have multiple wallets installed?");
+    }
+}
+
+window.addEventListener('load', setup);
+```
```

Comment:
````suggestion
```js title="index.js"
import detectEthereumProvider from "@metamask/detect-provider";

async function setup() {
  const provider = await detectEthereumProvider();

  if (provider && provider === window.ethereum) {
    console.log("MetaMask is available!");
    startApp(provider); // Initialize your dapp with MetaMask.
  } else {
    console.log("Please install MetaMask!");
  }
}

function startApp(provider) {
  if (provider !== window.ethereum) {
    console.error("Do you have multiple wallets installed?");
  }
}

window.addEventListener("load", setup);
```

````

Edited metadata, quotes, indents, and line break after snippet